dffml/util/net.py

```diff
@@ -406,17 +328,17 @@ def cached_download_unpack_archive(
     >>> import asyncio
     >>> from dffml import cached_download_unpack_archive
     >>>
-    >>> @cached_download_unpack_archive(
+    >>> async def files_in_dffml_commit_c4469a():
+    ...     dffml_dir = await cached_download_unpack_archive(
     ...     "https://github.com/intel/dffml/archive/c4469abfe6007a50144858d485537324046ff229.tar.gz",
     ...     "dffml.tar.gz",
     ...     "dffml",
     ...     "bb9bb47c4e6e4c6b7147bb3c000bc4069d69c0c77a3e560b69f476a78e6b5084adf5467ee83cbbcc47ba5a4a0696fdfc",
-    ... )
-    ... async def files_in_dffml_commit_c4469a(dffml_dir):
+    ...     )
     ...     return len(list(dffml_dir.rglob("**/*")))
     >>>
-    >>> asyncio.run(files_in_dffml_commit_c4469a())
-    124
+    >>> print(asyncio.run(files_in_dffml_commit_c4469a()))
```

Comment:
Same here, we can move the call to `cached_download_unpack_archive()` into the top level of the example, call with `asyncio.run()`, and then get rid of the function, just read the directory contents from the top level.